great_expectations/cli/init.py

```diff
@@ -117,29 +118,40 @@ def init(target_directory, view, usage_stats):
         else:
             datasources = context.list_datasources()
             if len(datasources) == 0:
-                datasource_name, data_source_type = add_datasource_impl(
-                    context, choose_one_data_asset=True
-                )
+                cli_message(SECTION_SEPARATOR)
+                if not click.confirm("Would you like to configure a Datasource?", default=True):
+                    cli_message("Okay, bye!")
+                    sys.exit(1)
+                datasource_name, data_source_type = add_datasource_impl(context, choose_one_data_asset=False)
                 if not datasource_name:  # no datasource was created
                     sys.exit(1)
 
             datasources = context.list_datasources()
             if len(datasources) == 1:
                 datasource_name = datasources[0]["name"]
 
-                success, suite_name = toolkit.create_expectation_suite(
-                    context,
-                    datasource_name=datasource_name,
-                    additional_batch_kwargs={"limit": 1000},
-                    open_docs=view,
-                )
-                if success:
-                    cli_message(
-                        "A new Expectation suite '{}' was added to your project".format(
-                            suite_name
-                        )
-                    )
+                cli_message(SECTION_SEPARATOR)
+                if not click.confirm("Would you like to profile new Expectations for a single data asset within your new Datasource?", default=True):
+                    cli_message("Okay, bye!")
```

Comment:
Instead of "Okay, bye!" let's provide some additional instructions:

"Okay, exiting now. To learn more about profilers, run great_expectations profile --help or visit docs.greatexpectations.io!"

Reply:
+1. Also, Profilers is a core concept, so please capitalize it.